src/routes/accounts.ts

```diff
@@ -21,17 +20,11 @@ export function accountsRouter({
   router.use(clientAuthMiddleware)
 
   const postFiatAccountRequestBodyValidator = (
-    req: express.Request<
-      {},
-      {},
-      PostFiatAccountRequestBody<SupportedFiatAccountSchemas>
-    >,
+    req: express.Request<{}, {}, PostFiatAccountRequestBody>,
     _res: express.Response,
     next: express.NextFunction,
   ) => {
-    req.body = validateSchema<
-      PostFiatAccountRequestBody<SupportedFiatAccountSchemas>
-    >(req.body, 'PostFiatAccountRequestBodySchema')
+    validateZodSchema(req.body, postFiatAccountRequestBodySchema)
```

Comment:
note that this doesn't limit validation to some subset of accepted schemas. I'm ok with dropping that feature, but open to ideas of how to preserve it.

Reply:
did this exist just as an example for any provider to restrict the schemas they support? Don't think there's any other option than manually constructing this with the required set of schemas https://github.com/fiatconnect/fiatconnect-types/blob/main/src/fiat-account.ts#L147-L171

Reply:
yeah I think so

Reply:
and yeah that's all I could think of, too